transit_odp/otc/weca/loaders.py

```diff
@@ -0,0 +1,33 @@
+from logging import getLogger
+
+from transit_odp.otc.models import Service
+from transit_odp.otc.weca.registry import Registry
+
+logger = getLogger(__name__)
+
+
+class Loader:
+    def __init__(self, registry: Registry):
+        self.registry: Registry = registry
+
+    def load(self):
+        logger.info("WECA job to refresh all the services started")
+        self.registry.process_services()
+        if len(self.registry.services) > 0:
+            self.delete_services()
+            self.load_services()
+
+        logger.info("WECA job finished the execution")
+
+    def load_services(self):
+        logger.info("Loading services into the database")
+        service_objects = []
+        self.registry.services.reset_index()
```

Comment:
need to pass inplace = true for persisting the  reseting index

Reply:
Updated